Sage/carbon
# docs(design-tokens): added examples for providers and moved docs to documentation section

### Proposed behaviour

Usage example for Token Providers components

### Current behaviour

Currently there are no examples of usage in Carbon components.

### Checklist




#### QA


### Additional context



### Testing instructions



Please look in JSDocs in `src/style/design-tokens/carbon-global-tokens-provider/carbon-global-tokens-provider.component.js`
and `src/style/design-tokens/carbon-scoped-story-provider/carbon-scoped-tokens-provider.component.js` files to see examples.

Also, please check Storybook Documentation If docs are located in right place in Documentation/Token Providers
